### PR TITLE
Mediation oriented api

### PR DESCRIPTION
**Pull request by samuelgoto**

### CODE REVIEW

File: mediation_oriented_api.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+# Benefits
+
+- Fewer prompts compared to the [Permission-oriented](consumers.md#the-permission-oriented-variation) variation
+- Prompts are "functional" (part of a decision the user has to make) rather than "informational" (speed bumps)
+- The browser has the ability to control the defaults
+
+# Challenges
+
+- Browsers need to become opinionated about sign-in and the subset of flows that enables it.
```
Comment (majido) [inline mediation_oriented_api.md:28]: nit: s/enables/enable

Comment (majido) [inline mediation_oriented_api.md:28]: Is ossification a separate challenge or something that follows from this?

**Review by majido — APPROVED** [review]
lgtm % nits

File: mediation_oriented_api.md (hunk 1 above)
Comment (samuelgoto) [reply mediation_oriented_api.md:28]: "enables" refers to the "subset" (not to the preprositional phrase "of flows") which is a singular?

Comment (samuelgoto) [reply mediation_oriented_api.md:28]: i think it is the same thing. i'll add a link to the terminology doc.

Comment (samuelgoto) [reply mediation_oriented_api.md:28]: ah, there isn't one ... i guess we should create an entry somewhere for ossification. will do that separately.

**Pull request merged**